streams/shared/src/main/scala/zio/stream/ZStream.scala

```diff
@@ -1773,26 +1771,26 @@ class ZStream[-R, +E, +A] private[stream] (private[stream] val structure: ZStrea
    * Enqueues elements of this stream into a queue. Stream failure and ending will also be
    * signalled.
    */
-  final def into[R1 <: R, E1 >: E, A1 >: A](
-    queue: ZQueue[R1, Nothing, Nothing, Any, Take[E1, A1], Any]
```

Comment:
Surely this one could stay, right? (though the result type can be `ZIO[R1, E, Unit]`)

Reply:
The method is staying. I don't think the additional type parameters add any flexibility and could cause type inference issues in some situations. We have a `ZStream[R, E, A]` and we can send it into any `ZQueue` that can accept values of type `Take[E, A]`. Because the input type accepted by the `Queue` is contravariant we can still send values to a `Queue` that can accept a supertype of `E` or `A`.

Reply:
I think that due to the covariance of `E` and `A` on `Take` and the covariance of the out type of `ZQueue`, these are redundant. Is this accurate @adamgfraser?

Reply:
Oops missed the comment from Adam. 👍